.gitignore

```diff
@@ -27,4 +26,4 @@ tutorials/recordings
 tutorials/cursor_image.png
 __pycache__/
 instance/
-mslib/idp/modules
+msidp/modules
```

Comment:
Maybe I missed a discussion about this, but why is this moved to a top-level directory? If this is meant for production it should stay in mslib/, if it is only for tests it should be in tests/, I think (I strongly favor moving it to tests/, since we should leave production for well-tested and mature IdP's like keycloak).

Reply:
Moved the below to the `mslib` directory based on the discussion in the weekly meeting 🙂